cmake/host-tools.cmake

```diff
@@ -38,7 +38,7 @@ else()
   # even after output is one line.
   message(STATUS "Found west: ${WEST} (found suitable version \"${west_version}\", minimum required is \"${MIN_WEST_VERSION}\")")
 
-  if (${west_version} VERSION_GREATER_EQUAL "0.7.1")
+  if (${west_version} VERSION_GREATER_EQUAL ${MIN_WEST_VERSION})
```

Comment:
This is always true isn't it? Due to us error-ing out above when it is false ?

Reply:
Yes, `FATAL_ERROR` does exit after printing the message. I'll remove the check.